prism_compile.c

```diff
@@ -993,16 +993,49 @@ pm_setup_args(pm_arguments_node_t *arguments_node, int *flags, struct rb_callinf
                       break;
                   }
                   else {
-                      *kw_arg = rb_xmalloc_mul_add(len, sizeof(VALUE), sizeof(struct rb_callinfo_kwarg));
-                      *flags = VM_CALL_KWARG;
-                      (*kw_arg)->keyword_len = (int) len;
+                    // We need to first figure out if all elements of the KeywordHashNode are AssocNodes
+                    // with static literal keys.
+                    // TODO: Figure this out from flags on the KeywordHashNode when Prism supports it
+                    int all_keys_static_literals = TRUE;
```

Comment:
I think this can be a boolean

Reply:
Sure! Updated. To be fair, back in my day, we didn't have `bool` as a type, so I was erring on the side of caution and falling back to old habits by using an `int` type 😄.